server_installation/topics/operator/keycloak-cr.adoc

```diff
@@ -29,7 +29,7 @@ ifeval::[{project_community}==true]
  or an Ingress for Kubernetes
 endif::[]
  for the {project_name} cluster.
-* `externalDatabase` - applies only if you want to connect an externally hosted database. That topic is covered in the xref:_external_database[external database] section of this guide.
+* `externalDatabase` - in order to connect to an externally hosted database. That topic is covered in the xref:_external_database[external database] section of this guide. This is enabled by default, only for testing purposes you can disable it and this will install an embedded PostgreSQL database.
```

Comment:
I'd also add somewhere around here a big warning box that using `externalDatabase: false` is not supported in prod envs.